src/components/CommentList/CommentList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './CommentList.scss';
+import PropTypes from 'prop-types';
+import { Comment } from '../Comment';
+
+export const CommentList = ({ list }) => (
+  <ul className="comment-list">
+    {list.map(comment => (
+      <li key={comment.id}>
+        <Comment commentItem={comment} />
```

Comment:
You can use spread operator here and destructuring in `Comment` component
```suggestion
        <Comment {...comment} />
```